packages/bits/demo/src/components/demo/wizard-v2/wizard-with-custom-icons/wizard-with-custom-icons-example.component.ts

```diff
@@ -1,6 +1,32 @@
 import { Component } from "@angular/core";
+import { IWizardConfig, IWizardStepStateIconConfig, WIZARD_CONFIG } from "@nova-ui/bits";
 @Component({
     selector: "nui-wizard-with-custom-icons-example",
     templateUrl: "./wizard-with-custom-icons-example.component.html",
+    providers: [
```

Comment:
Maybe a comment here (in addition to the one in the docs text) reminding consumers they can use this construction in an ngModule's `providers` property to propagate the wizard configuration to all wizards within a module.

Reply:
I can certainly do that!

The question I'm asking myself here is should we really remind Angular developers how to use angular DI in our component examples docs twice? I'm guessing you were thinking of people copying and pasting our examples without reading the docs description? In this case, they'd have that comment with a reminder in the examples code. Is my understanding valid?

Reply:
In Alpha's experience with user testing for nova charts, we observed people habitually copying/pasting code without looking at the text at all. So yeah, it's kind of a fail-safe to make sure they know the intended use case for the injection token. Maybe less experienced developers could benefit from it. Just a thought. Not a required change. :)